reporting/impl/src/main/resources/reports/templates/techReport-punchCard.ftl

```diff
@@ -143,23 +144,29 @@
                                             </#if>
                                         </#if>
                                         <#-- count: ${count?c}   max: ${maxForThisBox?c}   getLogaritmicDistribution(): ${ log?c } x 5 = ${ log * 5.0 } -->
-                                        <td class="circle size${ log?is_number?then((log * 5.0)?ceiling, "X")} sector sector${sectorTag.title}"><#-- The circle is put here by CSS :after --></td>
+                                        <td class="circle size${ log?is_number?then((log * 5.0)?ceiling, "X")} sector sector${sectorTag.title} table-tooltip" data-count="${countInteger?c}">
+                                            <#-- The circle is put here by CSS :after -->
+                                            <#if countInteger gt 0>
+                                            <span class="table-tooltiptext">${countInteger?c}</span>
+                                            </#if>
+                                        </td>
                                     </#if>
                                 <#else>
                                     <td>No technology sectors defined.</td>
                                 </#list>
                             </#list>
-                            <td class="sectorStats sector sizeMB">
+                            <td class="sectorStats sector sizeMB" data-count="${appProject.rootFileModel.retrieveSize()?c}">
```

Comment:
This seems to break the report creation during rules' testing.
Surrounding `appProject.rootFileModel.retrieveSize()` with `()` will fix the issue.

Reply:
I think this is fixed now.